scss/components/input-group.scss

```diff
@@ -17,6 +17,8 @@ $block: #{$fd-namespace}-input-group;
     vertical-align: bottom;
 
     //ELEMENTS *******************************************
+    //TO DO: REMOVE WHEN SELF CONTAINING INPUT GROUPS
+    $fd-elements-inputs--text: "input[type=text]", "input[type=password]", "input[type=email]", "input[type=url]", "input[type=search]", "input[type=tel]", "input[type=number]", "input[type=date]", "input[type=time]";
```

Comment:
This will be removed in the PR for input-group self contained styles, but is necessary for this file to function at the moment.